traffic_ops/testing/api/v5/deliveryservice_request_comments_test.go

```diff
@@ -178,7 +178,9 @@ func CreateTestDeliveryServiceRequestComments(t *testing.T) {
 		opts.QueryParameters.Set("xmlId", comment.XMLID)
 		resp, _, err := TOSession.GetDeliveryServiceRequests(opts)
 		assert.NoError(t, err, "Cannot get Delivery Service Request by XMLID '%s': %v - alerts: %+v", comment.XMLID, err, resp.Alerts)
-		assert.Equal(t, len(resp.Response), 1, "Found %d Delivery Service request by XMLID '%s, expected exactly one", len(resp.Response), comment.XMLID)
+		if !assert.Equal(t, len(resp.Response), 1, "Found %d Delivery Service request by XMLID '%s, expected exactly one", len(resp.Response), comment.XMLID) {
```

Comment:
Why continue and not error out?

Reply:
Because although creation of the currently processed comment has failed (or at best succeeded wrongly), other comments can still be created, which can help narrow down what the problem was and/or find other problems by continuing to run other tests.